src/main/java/pages/cartPage/shoppingCartTable/ShoppingCartTableRow.java

```diff
@@ -0,0 +1,115 @@
+package pages.cartPage.shoppingCartTable;
+
+import lombok.Getter;
+import lombok.Setter;
+import models.ProductModel;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.BasePage;
+import pages.cartPage.BaseTable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+
+@Getter
+@Setter
+public class ShoppingCartTableRow {
```

Comment:
Why we have part the row functionalities here and part in the shopping cart table?